### PR TITLE
Remove deprecated code

### PR DESCRIPTION
**Pull request by nemanja-boric-sociomantic**

### CODE REVIEW

**Comment by nemanja-boric-sociomantic** [comment]
@gavin-norman-sociomantic I'm not sure about this TODO: https://github.com/sociomantic-tsunami/swarm/pull/256/files#diff-e6b8af2086b1edd63b02934948bb9facR429

Do you care to advise or push a commit implementing this TODO?

**Comment by gavin-norman-sociomantic** [comment]
Pushed.

File: src/swarm/neo/node/ConnectionHandler.d [hunk 1]
```diff
@@ -207,19 +146,11 @@ class ConnectionHandler : IConnectionHandler
 
         public void initStats ( RequestStats request_stats )
         {
-            foreach ( code, rq; this.map )
-                if ( rq.name.length > 0 )
-                    request_stats.init(rq.name, rq.timing);
-
             foreach ( command, rq; this.request_info )
```
Comment (gavin-norman-sociomantic) [inline src/swarm/neo/node/ConnectionHandler.d:149]: Ooh, I think I just spotted a mistake here. `request_stats.init` only allows each request to be initialised once. `this.request_info` will contain one entry per _version_ of a request (i.e. potentially more than one entry per request).

Comment (gavin-norman-sociomantic) [reply src/swarm/neo/node/ConnectionHandler.d:149]: I'll make a PR against v4.x.x.

Comment (gavin-norman-sociomantic) [reply src/swarm/neo/node/ConnectionHandler.d:149]: https://github.com/sociomantic-tsunami/swarm/pull/257

**Review by gavin-norman-sociomantic — COMMENTED** [review]
We'll need to rebase v5.x.x again when https://github.com/sociomantic-tsunami/swarm/pull/257 is merged.

File: src/swarm/neo/node/ConnectionHandler.d (hunk 1 above)
Comment (nemanja-boric-sociomantic) [reply src/swarm/neo/node/ConnectionHandler.d:149]: Yeah, I'll think this PR will be fine we rebase v5.x.x on top of v4.x.x after that's merged.

**Comment by nemanja-boric-sociomantic** [comment]
Rebased

**Review by gavin-norman-sociomantic — APPROVED** [review]

**Pull request merged**